docs/src/domains/steps/dns.md

```diff
@@ -1,170 +1,139 @@
+    and configure your app and routes to [use the `www` subdomain as upstream](../../define-routes/_index.md).
+ 
+2.  Follow the instructions on [how to set up a custom domain](./_index.md).
+    When you come to configuring your DNS provider, instead of a `CNAME` record,
+    add an `A` record pointing from your domain to the redirection service.
+ 
+3.  Ensure that your redirects use the same protocol:
    `http://example.com` redirects to `http://www.example.com`, not to `https://www.example.com`.
-   Redirects from `http` to `https` are handled automatically.
-   Trying to change the protocol and domain in the same redirect causes issues for Let's Encrypt
-   and prevents the TLS certificate from being issued correctly.
-
-The extra redirect adds a few milliseconds to the first page load.
-
+    Redirects from `http` to `https` are handled automatically.
+    Changing the protocol and domain in the same redirect can prevent the [transport layer security (TLS) certificate](../../other/glossary.md#transport-layer-security-tls)
+    from being issued correctly.
+ 
 <--->
-
+ 
 +++
 title=Use `A` records
 file=none
 highlight=false
 +++
-
-If your registrar doesn't support custom records or domain forwarding and you can't use a redirection service, consider using `A` records.
-
-Using `A` records is _strongly discouraged_ and [should only be used as a last resort](#why-cname-records).
-
-Using `A` records has several limitations:
-
-* If the IPs change, you need to manually update your configuration.
-  Until you do, the site can appear offline because requests are lost.
-* Directly pointing at the edge routers bypasses their load-balancing functionality.
-  Should one of them go offline for maintenance (as happens periodically for upgrades),
-  about 1/3 of requests to your site are sent to the offline router and are lost, making the site appear offline.
-
-To configure your domain name to point to your project using `A` records:
-
-1. Get the IP addresses of your project's production environment by running `dig +short $(platform environment:info edge_hostname)`.
-2. Follow the instructions on [how to set up a custom domain](./_index.md).
-   When you come to configuring your DNS provider, replace the suggested `CNAME` record
-   with separate `A` records pointing from your domain to each of the IP addresses from step 1.
-   Incoming DNS lookups pick one of those IP addresses at random to use for the given request (known as round-robin DNS).
-
+ 
+If your DNS provider doesn't support any other workaround,
+consider using `A` records, but only as a last resort.
+ 
+When you use `A` records, [if a router's IP address changes](#cname-records),
+you need to update your `A` records manually.
+Until you do, your site can appear offline because requests are lost.
+ 
+To use `A` records, follow these steps:
+ 
+1.  To get the IP addresses of your project's production environment, run the following command:
+    ```bash
```

Comment:
:clipboard: 
````suggestion

        ```bash
````

Reply:
I applied this suggestion but ESLint then showed an error in my file (could be that my settings are off again? 🤔 ). How many spaces are we truly expecting here?